common/src/main/java/io/homeassistant/companion/android/common/data/repositories/BaseDaoWidgetRepository.kt

```diff
@@ -0,0 +1,25 @@
+package io.homeassistant.companion.android.common.data.repositories
```

Comment:
Please move this and all other files under the newly created `repositories` folder and place it under `main` to avoid adding this to Wear OS. The database code is fine to be here as both Wear OS and the main app share database code.

Reply:
When adding repositories, will add a layer where we can do the logic to interact with the database (no Android objects). So wear should also use this methods rather than the database Dao directly, as this makes easier to make the logic testable.